src/components/InternalDropdown/helpers/getValueNames.js

```diff
@@ -0,0 +1,10 @@
+export default function getValueNames(value) {
+    if (Array.isArray(value)) {
+        const names = value.map(v => v.name);
+        return names.filter(v => !!v);
```

Comment:
I that the options are going to be few and this code will run lightning-fast, but :) ... you are iterating twice ... you can use reduce here and make 1 iteration e.g.

```js
return value.reduce((seed, item) => {
    if (item.name) {
        seed.push(name);
    }
    return seed;
}, []);
```

Reply:
you are right, thanks.